templates/footer.twig

```diff
@@ -37,35 +37,38 @@
 			{% endif %}
 		</ul>
 
-		<ul class="list-unstyled footer-links">
-			{% for primary in footer_primary_menu %}
-				<li>
-					<a href="{{ primary.url }}"
-					   data-ga-category="Footer Navigation"
-					   data-ga-action="Primary Links"
-					   data-ga-label="{{ primary.title }}">
-						{{ primary.title }}
-					</a>
-				</li>
-			{% endfor %}
-		</ul>
+		{% if nav_type != 'planet4' %}
+			<ul class="list-unstyled footer-links">
+				{% for primary in footer_primary_menu %}
+					<li>
+						<a href="{{ primary.url }}"
+							data-ga-category="Footer Navigation"
```

Comment:
The current minimal implementation doesn't have the `data-ga` attributes in the social buttons. So, I assume we might add them.

Reply:
Yep, I also think they are supposed to have it. Maybe let Julia know they were missing here and we're adding them.